packages/api/src/external/carequality/mock-patient-discovery.ts

```diff
@@ -61,6 +58,5 @@ export async function mockResult(id: string, cxId: string, patientId: string) {
       ],
     },
   };
-  const patientDiscovery = patientDiscoveryResponseSchema.parse(body);
-  await createPatientDiscoveryResult(patientDiscovery);
+  await createPatientDiscoveryResult(body);
```

Comment:
I deleted this validation schema. On a previous PR (I think one of Jorges) we had agreed to not use zod schemas in the shared package

Reply:
schemas are back